filament/backend/src/webgpu/WebGPUDriver.cpp

```diff
@@ -864,8 +864,13 @@ void WebGPUDriver::draw2(uint32_t indexOffset, uint32_t indexCount, uint32_t ins
     mRenderPassEncoder.DrawIndexed(indexCount, instanceCount, indexOffset, 0, 0);
 }
 
-void WebGPUDriver::draw(PipelineState, Handle<HwRenderPrimitive>, uint32_t indexOffset,
+void WebGPUDriver::draw(PipelineState pipelineState, Handle<HwRenderPrimitive> rph, uint32_t indexOffset,
```

Comment:
nit: `rph` should be `renderPrimitiveHandle`, which I think is wha you're using here